### PR TITLE
WIP: Add user component

### PR DESCRIPTION
**Pull request by Ashaqra**

### CODE REVIEW

**Comment by geokhoury** [comment]
#### 📝Changed routes:


| [**/**](https://start-2p4viln13-josa.vercel.app/) |
|:-:|
|<a href="https://start-2p4viln13-josa.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/y9noUt2c7C9XOEN0kmIILWSRlAt9rmusdZ3f_vBjm4OnenMclnTb0cq5bdqy0ojWPCMGHZl3_TIZVMGbEqvXUQ.png">(view full size)</a> |







Commit 84135211e39e4dae1f75f16b7ce19088b7ca42cd (https://start-2p4viln13-josa.vercel.app).

File: components/User.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <div class="flex">
+      <img src="~/static/profile.png" class="user-image">
+      <p class="user-text">
+        Abdalla Shaqra
```
Comment (wissamataleh) [inline components/User.vue:6]: Use `props` with a default value instead

File: components/User.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      </p>
+    </div>
+    <div class="mt-2">
+      <p class="text-sm text-gray-300">
```
Comment (wissamataleh) [inline components/User.vue:10]: Replace with **`<a>`**

File: components/User.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        Logout
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'User'
```
Comment (wissamataleh) [inline components/User.vue:19]: * Change component name to `UserMenu`

File: components/User.vue [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+</script>
+
+<style scoped>
+.user-image {
+  width: 37px;
```
Comment (wissamataleh) [inline components/User.vue:25]: * Use `rem` instead of `px`

File: components/User.vue [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  height: 37px;
+}
+
+.user-text {
+  margin: 6px;
+  font-family: 'Courier New', Courier, monospace;
```
Comment (wissamataleh) [inline components/User.vue:31]: * Typography styles should only be in global CSS

File: components/User.vue [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'User'
+}
+</script>
+
+<style scoped>
```
Comment (wissamataleh) [inline components/User.vue:23]: * Use `TailwindCSS`

**Review by wissamataleh — CHANGES_REQUESTED** [review]
1- Add PR description.
2- Review comments.

File: components/UserMenu.vue [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+export default {
+  name: 'UserMenu',
+  props: {
+    userInfo: {
+      type: Object,
+      require: false,
+      default () {
+        return {
+          userName: 'JOSAUser',
+          userImage: '~/static/profile.png'
```
Comment (wissamataleh) [inline components/UserMenu.vue:27]: - Move image to static images folder

**Review by wissamataleh — CHANGES_REQUESTED** [review]

**Pull request merged**